Document/0x06h-Testing-Platform-Interaction.md

```diff
@@ -43,7 +51,7 @@ You should carefully validate any URL, before calling it. You can white-list app
 - CWE-939: Improper Authorization in Handler for Custom URL Scheme
 
 ##### Tools
-- Needle - https://labs.mwrinfosecurity.com/tools/needle/
```

Comment:
It is still relevant, but I found some difficulties in configuring Needle. IDB is easy to use and allows for customizing payload. Needle can be added in future as an alternative.